src/pages/blog.tsx

```diff
@@ -2,34 +2,39 @@ import { getSortedBlogMetadatas, Blog } from "../lib/blog-fetch";
 import { DateString } from "../components/date";
 import { Header } from "../components/header";
 import { Layout } from "../components/layout";
+import { Paper } from "../components/paper";
 import { NextPage, GetStaticProps } from "next";
 import Link from "next/link";
 import { FC } from "react";
-import { Card } from "../components/card";
+import styles from "../sass/pages/blog.module.sass";
 
 const BlogCard: FC<Blog> = ({ id, title, date }) => (
-  <>
-    <Link href={`/blog/${id}`}>
-      <a>
-        <Card>
-          <h3>{title}</h3>
-          <DateString dateString={date} />
-        </Card>
-      </a>
-    </Link>
-  </>
+  <Paper>
+    <img src="/alternative.png" className={styles.avatar} />
+    <div className={styles.cardText}>
+      <h3 className={styles.blogTitle}>{title}</h3>
+      <DateString dateString={date} />
+      <Link href={`/blog/${id}`}>
+        <div className={styles.linkWrapper}>
+          <a className={styles.pageLink}>記事を読む &rarr;</a>
+        </div>
```

Comment:
```suggestion
        <a className={styles.pageLink}>記事を読む &rarr;</a>
```
`blog.module.sass` で右寄せに揃える修正をすると、
`linkWrapper` の必要性がなくなる